test/unit/json_schema/specification_test.dart

```diff
@@ -179,6 +181,13 @@ void main() {
     draft9SkippedTestFiles,
     commonSkippedTests,
   );
+  runAllTestsForDraftX(
+    SchemaVersion.draft2019_09,
+    draft2019Format,
+    draft2019FormatSkippedTestFiles,
+    commonSkippedTests,
+    validateFormats: true,
+  );
```

Comment:
Since we need to operate with format validation enabled when running the optional format tests, that has to be done separately. :cry: